project/jsonld/sssom_schema.jsonld

```diff
@@ -1844,7 +1844,7 @@
     {
       "name": "similarity_score",
       "definition_uri": "https://w3id.org/sssom/similarity_score",
-      "description": "A score between 0 and 1 to denote the similarity between two entities, where 1 denotes equivalence, and 0 denotes disjointness. The score is meant to be used in conjunction with the similarity_measure field, to document, for example, the lexical or semantic match of a matching algorithm.",
+      "description": "A score between 0 and 1 to denote the similarity between two entities, where 1 denotes equivalence, and 0 denotes disjointedness. The score is meant to be used in conjunction with the similarity_measure field, to document, for example, the lexical or semantic match of a matching algorithm.",
```

Comment:
`disjointness` is a technical term in our domain, and should not be replaced.

Reply:
indeed... I even considered to file fixup to codespell, but decided to not bother for now.  I will add a skip of this word. Thanks for spotting.